flink-shaded-hadoop/flink-shaded-yarn-tests/src/main/resources/META-INF/NOTICE

```diff
@@ -0,0 +1,47 @@
+flink-shaded-yarn-tests
+Copyright 2014-2018 The Apache Software Foundation
+
+This product includes software developed at
+The Apache Software Foundation (http://www.apache.org/).
+
+This project bundles the following dependencies under the Apache Software License 2.0 (http://www.apache.org/licenses/LICENSE-2.0.txt)
```

Comment:
I will have to scan over my PR again if we really want to include all ASL dependencies.